cylc/flow/scripts/lint.py

```diff
@@ -265,6 +265,10 @@ def check_indentation(line: str) -> bool:
     return bool(len(match[0]) % 4 != 0)
 
 
+CHECK_FOR_OLD_VARS = re.compile(
+    r'CYLC_VERSION\s*=\s*\{\{CYLC_VERSION\}\}|ROSE_VERSION\s*='
+    r'\s*\{\{ROSE_VERSION\}\}|FCM_VERSION\s*=\s*\{\{FCM_VERSION\}\}')
```

Comment:
Should tolerate arbitrary whitespace inside the `{{` too.

Reply:
Yes!